Added EDRReader information (#230)
MDAnalysis/UserGuide: doc/source/formats/auxiliary.rst

    u2 = mda.Universe(PDB, TRR)
    for desc in descriptions:
        u2.trajectory.add_auxiliary(**desc)

EDR Files
---------

> EDR files are created by GROMACS during simulations and contain additional time-series non-trajectory data on the

Reviewer: ```suggestion
EDR files are created by GROMACS during simulations and contain additional non-trajectory time-series data of the
```
?

system, such as energies, temperature, or pressure. The EDRReader allows direct reading of these binary files
and associating of the data with trajectory time steps just like the XVGReader does. As all functionality of the base AuxReaders (see XVGReader above) also works with
the EDRReader, this section will highlight functionality unique to the EDRReader.

Standalone Usage
----------------

> The EDRReader is initialised by passing the path to an EDR file to it.

Reviewer: ```suggestion
The EDRReader is initialised by passing the path to an EDR file as an argument.
```
"to it" seems like a bit of a stranged personification? might just be me

> The EDRReader is initialised by passing the path to an EDR file as an argument.

Reviewer 2: Show an example (possibly commented out) with a normal path also rather than just a filename-handle from the testsuite as this will not be as familiar to people.

Author: Added, thank you!

.. ipython:: python

    temp = aux.get_data("Temperature")
    print(temp.keys())
    energies = aux.get_data(["Potential", "Kinetic En."])
    print(energies.keys())
    all_data = aux.get_data()
    print(all_data.keys())

The times of each data point are always part of the returned dictionary to facilicate plotting.

.. ipython:: python

    import matplotlib.pyplot as plt
    plt.plot(temp["Time"], temp["Temperature"])
    plt.ylabel("Temperature [K]")
    plt.xlabel("Time [ps]")
    plt.show()

.. plot::

    import matplotlib.pyplot as plt
    import MDAnalysis as mda
    from MDAnalysisTests.datafiles import AUX_EDR
    aux = mda.auxiliary.EDR.EDRReader(AUX_EDR)
    temp = aux.get_data("Temperature")
    plt.plot(temp["Time"], temp["Temperature"])
    plt.ylabel("Temperature [K]")
    plt.xlabel("Time [ps]")
    plt.show()

Unit Handling
-------------

On object creation, the EDRReader creates a ``unit_dict`` attribute which contains
information on the units of the data stored within. These units are read from the
EDR file automatically and by default converted to MDAnalysis base units where such unit

> is defined. The automatic unit conversion can be disabled by setting the ``convert_units`` kwarg to False.

Reviewer: ```suggestion
are defined. The automatic unit conversion can be disabled by setting the ``convert_units`` kwarg to False.
```
?
